src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200211122058_RemoveRedundantProcs.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.IO;
+using System.Reflection;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class RemoveRedundantProcs : Migration
+    {
+        private const string MigrationsPath = "Migrations";
+        private const string MigrationId = "20200211122058";
+        private const string OrigMigrationId = "20200103101609";
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            // Types
+            ExecuteFile(migrationBuilder, $"{MigrationId}_RemoveRedundantProcs_down.sql");
+            // Routines
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertFilter.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertFilterFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertFilterGroup.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertFilterGroupFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertFilterItem.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertFilterItemFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertIndicator.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertIndicatorFootnote.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertIndicatorGroup.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertRelease.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertSubject.sql");
+            ExecuteFile(migrationBuilder, $"{OrigMigrationId}_Routine_UpsertSubjectFootnote.sql");
+        }
+
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Routines
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilter");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterGroup");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterGroupFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterItem");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertFilterItemFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertIndicator");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertIndicatorFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertIndicatorGroup");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertSubject");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertSubjectFootnote");
+            migrationBuilder.Sql("DROP PROCEDURE dbo.UpsertRelease");
```

Comment:
This isn't the full list but I'm presuming that's intentional? The following remain:

- UpsertFootnote,
- UpsertTheme
- UpsertTopic
- UpsertPublication
- UpsertLocation

Reply:
Yes intentional - only needed for the tables where I need to upsert - the release is deleted on re-run so all related tables can just use standard data factory copy.

Reply:
Great thanks